.github/workflows/ci-cd-automation.yml

```
@@ -0,0 +1,27 @@
+name: Servicefoundry CI/CD Automation Example
+
+on:
+  push:
+    paths:
+      - 'ci-cd-automation/*'
+
+jobs:
+  production_deploy:
+    runs-on: ubuntu-latest
+    env:
+      TFY_HOST: https://app.develop.truefoundry.tech
```

Not exactly sure about this one, I think we should keep this to production
```suggestion
      TFY_HOST: https://app.truefoundry.com/
```

---

yup, I don't know whose account to use to deploy there